bokeh/core/property/numeric.py

```diff
@@ -269,6 +271,14 @@ class Angle(Float):
     '''
     pass
 
+class Base64String(String):
```

Comment:
This is in the wrong module, this should be in `bokeh/core/property/string.py`

Reply:
At the moment, I have moved the class from `numeric.py` to `regex.py`, but have not renamed the file. There might be various places where regex.py file is currently used and renaming it would cause a problem. How should I tackle this? Also could you please elaborate on,

> we would need to leave a deprecated regex.py as well, to maintain back compat

Reply:
@ayushiitr2901 Right I am suggesting renaming `regex.py` to `string.py` but then also adding a new `regex.py` that does nothin except import the `Regex` property (to preserve compatibility) and also calls `deprecated(...)` to note that the`regex` module is deprecated.